kubejs/client_scripts/jei_hide.js

```diff
@@ -49,6 +48,37 @@ JEIEvents.hideItems(event => {
 	]
 	backpack.forEach(element => event.hide(element))
 
+	const storages = [
+		'sophisticatedstorage:advanced_feeding_upgrade',
+		'sophisticatedstorage:advanced_magnet_upgrade',
+		'sophisticatedstorage:auto_blasting_upgrade',
+		'sophisticatedstorage:auto_smelting_upgrade',
+		'sophisticatedstorage:auto_smoking_upgrade',
+		'sophisticatedstorage:blasting_upgrade',
+		'sophisticatedstorage:chipped/alchemy_bench_upgrade',
+		'sophisticatedstorage:chipped/botanist_workbench_upgrade',
+		'sophisticatedstorage:chipped/carpenters_table_upgrade',
+		'sophisticatedstorage:chipped/glassblower_upgrade',
+		'sophisticatedstorage:chipped/loom_table_upgrade',
+		'sophisticatedstorage:chipped/mason_table_upgrade',
+		'sophisticatedstorage:chipped/tinkering_table_upgrade',

```

Comment:
These chipped upgrades are all very useful for building. In general we don't want to nerf building by removing things like this. I see you still have the recipes for the chipped backpack upgrades, so maybe make similar recipes for the chipped storage upgrades.